src/test/resources/application.yml

```diff
@@ -153,3 +153,6 @@ akhq:
           groups:
             - operator
 
+  retry:
```

Comment:
Please let me know if this is okay to add - I've added this so we can control the delay when running unit tests, but I'm aslo wary we're exposing this property to users.

Reply:
Exposing the options is fine I think but please puyt in `akhq.topic.retry` and add it to `application.example.yml`